### PR TITLE
Release v0.6 docs

### PR DESCRIPTION
**Pull request by vincentpierre**

### CODE REVIEW

**Comment by xiaomaogy** [comment]
We still have more than 5 places in the doc where we have .bytes file mentioned. 

eg:

In Getting-Started-with-Balance-Ball.md, Learning-Environment-Executable.md, Training-Imitation-Learning.md.

**Comment by xiaomaogy** [comment]
In test_demo_loader.py and test_trainer_controller.py, we still have Ball3DBrain mentioned.

File: docs/Basic-Guide.md [hunk 1]
```diff
@@ -200,24 +201,24 @@ INFO:mlagents.envs:Hyperparameters for the PPO Trainer of brain Ball3DBrain:
...
+INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 9000. Mean Reward: 18.151. Std of Reward: 16.871. Training.
+INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 10000. Mean Reward: 27.284. Std of Reward: 28.667. Training.
 ```
 
 ### After training
 
 You can press Ctrl+C to stop the training, and your trained model will be at
-`models/<run-identifier>/editor_<academy_name>_<run-identifier>.bytes` where
-`<academy_name>` is the name of the Academy GameObject in the current scene.
+`models/<run-identifier>/<brain_name>.tf` where
```
Comment (xiaomaogy) [inline docs/Basic-Guide.md]: Why do we no longer put editor before the .tf file name?

File: docs/Learning-Environment-Design-Learning-Brains.md [hunk 2]
```diff
@@ -45,16 +45,16 @@ To use a graph model:
...
+   program. (Where `model_name` is the name of the model file, which is
    constructed from the name of your Unity environment executable and the run-id
    value you assigned when running the training process.)
 
    You can
    [import assets into Unity](https://docs.unity3d.com/Manual/ImportingAssets.html)
    in various ways. The easiest way is to simply drag the file into the
    **Project** window and drop it into an appropriate folder.
-4. Once the `environment.bytes` file is imported, drag it from the **Project**
+4. Once the `model_name .tf` file is imported, drag it from the **Project**
```
Comment (xiaomaogy) [inline docs/Learning-Environment-Design-Learning-Brains.md]: Extra space here.

**Comment by xiaomaogy** [comment]
In Using-TensorFlow-Sharp-in-Unity.md we still mention 'External Brain' in some places.

File: docs/Basic-Guide.md (hunk 1 above)
Comment (vincentpierre) [reply docs/Basic-Guide.md]: We use the brain_name directly

**Comment by xiaomaogy** [comment]
In test_demo_loader.py and test_trainer_controller.py, we still have Ball3DBrain mentioned.

This is still not resolved. 

@vincentpierre

**Comment by xiaomaogy** [comment]
Once that's fixed it is good to be merged.

**Comment by vincentpierre** [comment]
test_demo_loader.py is not related to documentation. It is just for tests.

**Comment by vincentpierre** [comment]
The new docs refer to `Inference-Engine.md` that is not pushed. File can be seen [here](https://docs.google.com/document/d/1txCzy3M7p4d_sC2JaBonx5-5v7il8G4UnT9b2mUz5Eg/edit)

**Pull request merged**